src/css/components/_c-banner.scss

```diff
@@ -2,11 +2,11 @@ $_banner-triangle-size: 1.49rem;
 
 
 .c-banner {
-	background-color: $color-green-tint;
-	background-image: url("#{$path-to-img}/resource-banner-inset.svg");
+	@include var(background-color, card-accent);
+	background-image: url("#{$path-to-img}/resource-banner-inset-lightmode.svg");
```

Comment:
I think this needs to switch between lightmode and darkmode. Right now it's always in lightmode and so in darkmode it doesn't look the way it should be:

![image](https://user-images.githubusercontent.com/9530293/162786744-3be8cbaa-be2e-4f34-83ad-4c31b2104307.png)